### DISCUSSION TITLE
Allow a compass to point to a home when set

### DISCUSSION THREAD

**Discussion by EMcKee10** [Zrips/CMI#2346] [Feature Requests]
**Description of feature/improvement request:**
When doing /compass (targetName) (sourceName) (x) (z) (worldname) (reset) (-s) maybe add a extra variable for any homes a person might have. 
---
### Detailed explanation of what it should do, when and in what situation: ###
so lets say a player named Alex wants to be navigated to a home they set called "StevesHouse" with a addition parameter (and the additional logic) the player typing "/compass home:StevesHouse alex" would have the compass point towards the home set named StevesHouse. obviously this would be on a player to player basis, so Alex has his home set named StevesHouse, but Steve may not, and even if he did it still wouldnt work for him unless he had his compass set to StevesHouse to after naming his house set "StevesHouse"

### Basic explanation as to why this feature is needed and/or useful to have: ###
This would allow even more functionality to compas, with the added benefit of allowing servers to have a reason to have /sethome and /delhome but not have /home. Survival servers might really enjoy this since /home isnt always wanted, but players want to know where their home is if they are ever lost. and honest, who uses the ingame compass (not the command) anyway. 
In addition to above, this would also allow players to reset their compass to their house, as /compass reset, only resets the compass to the spawn point, not their bed point.

**Comment by TheBeast2155** [comment]
+1
i will use it too :) will be nice to like it to /homes to :)
